### ISSUE TITLE
[Feature request] Add the ability to change the start and and end times of a clip

### ISSUE THREAD

**Issue by Sosenteam** [ShaneIsrael/fireshare#259]
I sometimes crop my clips to the cool parts,  but I think that this could be a simple-ish feature to add assuming you don't actually edit the clips, but only the metadata/playback.

**Comment by Acrosicious** [comment]
Agree, would be nice to just create clips with the frontend interface. I would even suggest creating actual clips, as in, multiple clips per video.
